packages/imperative/src/utilities/src/CliUtils.ts

```diff
@@ -378,20 +378,26 @@ export class CliUtils {
      * @memberof CliUtils
      */
     public static showMsgWhenDeprecated(handlerParms: IHandlerParameters) {
-        if (handlerParms.definition.deprecatedReplacement) {
+        let oldCmd: string;
+        if (handlerParms.definition.deprecatedReplacement || handlerParms.definition.deprecatedReplacement === "") {
             // form the command that is deprecated
-            let oldCmd: string;
             if (handlerParms.positionals.length >= 1) {
                 oldCmd = handlerParms.positionals[0];
             }
             if (handlerParms.positionals.length >= 2) {
                 oldCmd = oldCmd + " " + handlerParms.positionals[1];
             }
```

Comment:
If a single command (rather than a command group) is deprecated, the message may not show the full command. For example, I deprecated the "zowe zcsp issue cwp" command in the CLI sample plug-in and it shows the following message:
```
Warning: The command 'zcsp issue' is deprecated.
Obsolete component. No replacement exists
```

If we show all positionals, then the full command name will be displayed (e.g. `zowe zcsp issue cwp`):
```suggestion
            const oldCmd = handlerParms.positionals.join(" ");
```

Reply:
I think that bug may have been there all along, good to know though. Thanks @t1m0thyj